noxfile.py

```diff
@@ -0,0 +1,82 @@
+import json
+from argparse import ArgumentParser
+from enum import Enum
+from pathlib import Path
+from typing import List, Optional
+
+import yaml
+from pydantic import BaseModel
+
+import nox
+
+ROOT = Path(__file__).parent
+
+# default actions to be run if nothing is explicitly specified with the -s option
+nox.options.sessions = []
+
+
+class TestStatus(Enum):
+    stable = "stable"
+    unstable = "unstable"
+
+
+class TestClassification(Enum):
+    normal = "normal"
+    large = "large"
+
+
+class WipStatus(Enum):
+    wip = "wip"
+    no_wip = "no-wip"
+
+
+class NBTestBackend(Enum):
+    on_prem = "onprem"
+    saas = "saas"
+    empty = ""
+
+
+class NBTestDescription(BaseModel):
+    """
+    Description of a notebook test. 
+    """
+    name: str
+    test_file: str
+    test_backend: NBTestBackend
+    wip: WipStatus
+
+class TestList(BaseModel):
+    tests: Optional[List[NBTestDescription]]
+
+class TestSets(BaseModel):
+    stable: TestList
+    unstable: TestList
+
+class TestRepository(BaseModel):
+    normal: TestSets
+    large: TestSets
+
+def _load_test_repository() -> TestRepository:
+    yaml_file_path = Path('nb_tests.yaml')
+    with open(yaml_file_path, 'r') as file:
+        yaml_data = yaml.safe_load(file)
+        return TestRepository(**yaml_data)
+
+@nox.session(name="get-notebook-tests", python=False)
+def get_notebook_tests(session: nox.Session):
+    """
+    Validates the correctness of the given tag.
+    """
+    test_status_values = [ts.value for ts in TestStatus]
+    test_classification_values = [ts.value for ts in TestClassification]
+
+    parser = ArgumentParser(
+        usage=f"nox -s {session.name} -- --test-status {{{','.join(test_status_values)}}} <--test-classification {{{','.join(test_classification_values)}}}>")
```

Comment:
That is not readable. I recommend extracting the joins. Having the {{ }} is not helpful to understand this. Either use a different character or assemble it outside and maybe without f-string. Furthermore, `<--test-classifcation` incorrect